server/controllers/api/assessmentDayDataController/index.js

```diff
@@ -73,7 +73,17 @@ const AssessmentDayDataController = {
           await SiteMetadataModel.upsert(
             appDb,
             { study },
-            { $addToSet: { participants: participant } }
+            {
+              $addToSet: {
+                participants: {
+                  Active: 1,
+                  Consent: new Date(),
```

Comment:
Should this be a date from the metadata, or do we track consent based on upload date?